### ISSUE TITLE
Unable to open koreader on PocketBook840 on version 2020.10 and higher

### ISSUE THREAD

**Issue by pavoldecky** [koreader/koreader#6889]
* KOReader version: 2020.10, 2020.11
* Device: PocketBook840, Inkpad

#### Issue
After an upgrade or clean install from version 2020.09 to 2020.10 or 2020.11 I cannot open koreader on PocketBook840. When I open koreader, I can see the pocketbook loading indicator for one second, after this screen is refreshed, but nothing happens. crash.log was not created and I cannot find any other helpful logs. For me, it looks like this is somehow related to changes for Pocketbook in version 2020.10 as the latest working version is 2020.09.

Can you please somehow help me, how can I turn on some additional logging so I can provide more information? 

Thanks for your help

#### Steps to reproduce
1. Install latest version 2020.11 to PocketBook840
2. Unable to open koreader app,

##### `crash.log` (if applicable)
`crash.log` was not created

**Comment by pazos** [comment]
@avaxxx: do you use the icon to launch KOReader or do you open a book from PB library?. If the later then do the former. If a log is still not created then you'll need to [get shell access](https://www.mobileread.com/forums/showthread.php?t=325185) and execute `/mnt/ext1/applications/koreader.app` from there, to see what's happening.

**Comment by avaxxx** [comment]
@pazos I am trying to open koreader with icon and also with a book in PB library, both steps are not working. I think I can have maybe some issue mostly with PocketBook itself. I am using quite old firmware 5.4.98, because when I bought it, firmware 5.15 caused freezing, so I downgraded to 5.4 FW and everything was OK. Today I tried to update to latest 5.20 FW, but afterwards, PB was in the boot loop, so I had to again install manually 5.4 FW.

Anyway, I tried to get shell access with [pbjb](https://github.com/ezdiy/pbjb/releases/tag/v8) you mentioned, but without success, I ran Jailbreak app, but nothing happened. According to the creator of this app, this is working with the latest 5.20 FW which I don't have. Afterward I tried [pbsshd](https://www.mobileread.com/forums/showthread.php?t=159636) but again with no success :( 

Finally I tried [pbterm](https://github.com/Alastor27/pbterm) and I was able to execute `/mnt/ext1/applications/koreader.app` there and here is the result

```
/mnt/ext1/applications/koreader.app: line 18: /mnt/secure/su: not found
/mnt/ext1/applications/koreader.app: line 32: syntax error: you disabled math support for $((arith)) syntax
```

Also, I found in another issue here in GitHub mentioning strace cmd so here is result, maybe it can help - [strace.txt](https://github.com/koreader/koreader/files/5555130/strace.txt)
